apps/site/pages/docs/getting-started/project-structure.mdx

```diff
@@ -12,4 +12,72 @@ import { Callout } from 'nextra-theme-docs'
 </header>
 <Callout type="warning" emoji="⚠️">
 This documentation is currently under development.
-</Callout>
\ No newline at end of file
+</Callout>
+
+In the previous guide of this track, you started your first FastStore store, learned how to run a local server and make your first change in the storefront. Now it's time to understand how our project is structured.
+
+By understanding the structure of a FastStore project, you can better navigate the codebase and make customizations to fit your specific needs.
+
+Let's start taking a look at the structure that represents the source code of a FastStore project:
+  
+  ```
```

Comment:
should we list `.faststore` here? 🤔

Reply:
Also, by adding the `.faststore`, the doc will remain relevant and understandable even if someone does not access the doc through this track.

Reply:
I'd actually refrain from listing .github, .gitignore, and node_modules as they are not related to FastStore.